src/lbaf/IO/lbsMeshWriter.py

```diff
@@ -0,0 +1,306 @@
+from logging import Logger
+import os
+import sys
+import math
+import numbers
+import random
+import vtk
+
+from .lbsGridStreamer import GridStreamer
+from ..Model.lbsPhase import Phase
+
+
+class MeshWriter:
+    """A class to write LBAF results to mesh files via VTK layer
+    """
+
+    def __init__(self, p: Phase, grid_size, object_jitter: float, f="lbs_out", r=1., output_dir=None, logger: Logger = None):
+        """ Class constructor:
+            p: Phase instance
+            grid_size: iterable containing grid sizes in each dimension
+            object_jitter: coefficient of random jitter with magnitude < 1
+            f: file name stem
+            r: grid_resolution value
+            output_dir: output directory
+        """
```

Comment:
Doc strings quotations does not correspond the the one changes in other files.

Reply:
Should be all the same now in this file